packages/modules/src/SessionKeyManagerModule.ts

```diff
@@ -49,7 +52,9 @@ export class SessionKeyManagerModule extends BaseValidationModule {
     } else if (moduleConfig.version) {
       const moduleAddr = SESSION_MANAGER_MODULE_ADDRESSES_BY_VERSION[moduleConfig.version]
       if (!moduleAddr) {
-        throw new Error(`Invalid version ${moduleConfig.version}`)
+        // throw new Error(`Invalid version ${moduleConfig.version}`)
+        Logger.error(`Invalid version ${moduleConfig.version}`)
```

Comment:
Should keep throwing the error in case invalid version is passed.

Reply:
even if version is invalid I was thinking to use default module address then which comes from the constants.

Reply:
fixing by keeping it in the else part. do review again for final